op_builder/builder.py

```diff
@@ -839,7 +839,9 @@ def cxx_args(self):
 
         CPU_ARCH = self.cpu_arch()
         SIMD_WIDTH = self.simd_width()
-        CUDA_ENABLE = self.is_cuda_enable()
+        CUDA_ENABLE = (
+            "-D__DISABLE_CUDA__" if self.is_rocm_pytorch() else self.is_cuda_enable()
```

Comment:
Thanks for this PR. Can you please help with the following changes to improve the codebase beyond your fix?

1. Rename the method `is_cuda_enable()` -> to `get_cuda_compile_flag()`, which is more meaningful. 
2. Handle `self.is_rocm_pytorch()` case in `get_cuda_compile_flag()`, returning `-D__DISABLE_CUDA__` as appropriate. 

Are you able to help these changes? Thanks!

Reply:
fixed